test/metadata/main.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/amp-labs/connectors"
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/salesforce"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	creds, err := os.ReadFile("../creds.json")
+	if err != nil {
+		slog.Error("Error opening creds.json", "error", err)
+		return
+	}
+
+	var credsMap map[string]interface{}
+
+	if err := json.Unmarshal(creds, &credsMap); err != nil {
```

Comment:
Chris and I discussed on this. I think it can be better addressed in separate PR to consolidate ways to get credentials. 

Created task https://linear.app/ampersand/issue/ENG-522/consolidate-reading-credentials-from-testing-runs-in-connectors